pkg/apis/core/field_constants.go

```diff
@@ -27,6 +27,10 @@ const (
 	// the BackupBucket for a core.gardener.cloud/v1beta1 BackupEntry.
 	BackupEntryBucketName = "spec.bucketName"
 
+	// BastionSeedName is the field selector path for finding
+	// the Seed cluster of a operations.gardener.cloud/v1alpha1 Bastion.
+	BastionSeedName = "spec.seedName"
+
```

Comment:
You can use:
https://github.com/gardener/gardener/blob/0e22d80c5945368d0decd51163a6b146b21539fa/pkg/apis/operations/field_constants.go#L20-L22